share/doc/homebrew/Prose-Style-Guidelines.md

```diff
@@ -0,0 +1,73 @@
+# Prose Style Guidelines
+
+This is a set of style and usage guidelines for Homebrew's prose documentation aimed at users, contributors, and maintainers (as opposed to executable computer code). It applies to documents like those in `share/doc/homebrew` in the `Homebrew/brew` repo, announcement emails, and other communications with the Homebrew community.
+
+This does not apply to any Ruby or other computer code. You can use it to inform technical documentation extracted from computer code, like embedded man pages, but it's just a suggestion there.
+
+## Goals and audience
+
+The primary goal of Homebrew's prose documents is communicating with its community of users and contributors. "Users" includes "contributors" here; wherever you see "users" you can substitute "users and contributors".
+
+Understandability trumps any particular style guideline.
+
+Users trump maintainers, except in specifically maintainer-focused documents.
+
+Homebrew's audience includes users with a wide range of education and experience, and users for whom English is not a native language. We aim to support as many of those users as feasible.
+
+We strive for "correct" but not "fancy" usage.
+
+This is a set of guidelines to be applied using human judgment, not a set of hard and fast rules. It is like [The Economist's Style Guide](http://www.economist.com/styleguide/introduction) or [Garner's Modern American Usage](https://en.wikipedia.org/wiki/Garner's_Modern_American_Usage). It is less like the [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide). All guidelines here are open to interpretation and discussion. 100% conformance to these guidelines is *not* a goal.
+
+The intent of this document is to help authors make decisions about clarity, style, and consistency. It is not to help settle arguments about who knows English better. Don't use this document to be a jerk.
+
+## Guidelines
+
+We prefer:
+
+### Style and usage
+
+* British/Commonwealth English over American English, in general
+* Structural/syntactic punctuation, not delivery-pacing punctuation
```

Comment:
It's probably not just being non-native-English; the "delivery-pacing" view is a minority, descriptive view. The "structural/syntactic" approach is that punctuation is part of syntax, and that the correct punctuation is determined by the written language itself, mostly by grammatical rules. The "delivery-pacing" approach is that punctuation – commas, semicolons, periods – is more a guide to how you would read and deliver the prose if you were speaking out loud, and that using punctuation to indicate cadence is a lot more informal but still legitimate.

I probably wouldn't have mentioned this, except some of the existing Homebrew documentation seems to follow the more informal delivery-pacing approach.

We should probably just delete this point altogether. Everyone probably assumes "structural/syntactic", and making it a point of discussion is probably just a distraction.